### PR TITLE
Feat/auth footer

### PR DESCRIPTION
**Pull request by Aditi0505**
# Change Logs:

## Description:

- Added Auth pages
- Added Footer
- Fixed correct category fetch on api call.

## Screenshot 
image
image

## Preview Link:
https://deploy-preview-8--guessthisquiz.netlify.app/

### CODE REVIEW

File: src/pages/Auth/Login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                name="checkbox"
+                id="checkbox"
+                className="inputBox margin-tb-sm"
+              />
+              <label
+                htmlFor="remember"
+                className="label-content text-sm padding-xs"
+              >
+                Remember me
+              </label>
```
Comment (Shailseen) [inline src/pages/Auth/Login/Login.jsx:44]: You can take input `name` same as label `htmlfor ` .So user when click on label then input box will be checked.

File: src/pages/Auth/Signup/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                name="checkbox"
+                id="checkbox"
+                className="inputBox margin-tb-sm"
+              />
+              <label
+                htmlFor="remember"
+                className="label-content text-sm padding-xs"
+              >
+                I accept all terms and conditions
+              </label>
```
Comment (Shailseen) [inline src/pages/Auth/Signup/Signup.jsx:42]: Same as above.

**Review by rohanmathur91 — APPROVED** [review]
- Good folder structure and clean coding
 - Overall looks good, great work!

File: src/pages/Quiz/Quiz.jsx [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import { Link } from "react-router-dom";
 import { Question } from "../../components/Question/Question";
 import { useCurrentQuestion, useQuestion } from "../../context";
```
Comment (Shailseen) [inline src/pages/Quiz/Quiz.jsx:3]: same as above using absolute path

**Review by Shailseen — APPROVED** [review]
- Some suggestion for make changes
- Ui looks good and also folder structure is well maintained.

File: src/components/Card/Card.jsx [hunk 4]
```diff
@@ -1,20 +1,27 @@
-import { useEffect } from "react";
-import { Link } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
 import { useQuestion } from "../../context";
 import { fetchQuestionsByCategory } from "../../services";
```
[the remark this answers is not in the document]
Comment (Aditi0505) [reply src/components/Card/Card.jsx:3]: Sure, I will implement this later on.

File: src/pages/Auth/Login/Login.jsx (hunk 1 above)
Comment (Aditi0505) [reply src/pages/Auth/Login/Login.jsx:44]: Sure, I will implement this later on, while working on auth functionality.

**Pull request merged**